cmd/containerd/main.go

```diff
@@ -94,6 +95,16 @@ func main() {
 		if err := applyFlags(context, config); err != nil {
 			return err
 		}
+		// cleanup temp mounts
+		t, err := mount.TempLocation(filepath.Join(config.Root, "tmpmounts"))
+		if err != nil {
+			return errors.Wrap(err, "creating temp mount location")
+		}
+		mount.DefaultTempLocation = t
+		// unmount all temp mounts on boot for the server
+		if err := t.Unmount(0); err != nil {
```

Comment:
I think we should have debug log prints around here for helping users analyze potential Umount hangup.

Also, maybe we should panic on timeout